README.md

```diff
@@ -1,6 +1,64 @@
 # About Power Flows DMN Model
 Power Flows DMN Model
 
+# DMN model
+Power Flows model has been designed as a YAML file. The file contains information about input and output data. The additional division is sections with fields and decisions.
+
+```yaml
+id: sample_decision_id
+name: Sample Decision Name
+hit-policy: UNIQUE
+expression-type: groovy
+fields:
+  in:
+    age:
+      expression-type: feel
```

Comment:
the actual input evaluation is missing here.
I think we should default to evaluating name as expression in execution context, resulting in expression like ${age}
but we should document a more verbose example like
```yaml
    age:
      expression-type: feel
      expression: toYear(now()) - toYear(birthDate)
      type: integer
      description: This is something about age
```